Description: Use org.jvnet.jaxb2.maven2:maven-jaxb2-plugin instead of org.codehaus.mojo:jaxb2-maven-plugin
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/engine/pom.xml
+++ b/engine/pom.xml
@@ -136,6 +136,34 @@
         </testResources>
         <plugins>
             <plugin>
+                <groupId>org.jvnet.jaxb2.maven2</groupId>
+                <artifactId>maven-jaxb22-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>jaxb-generate</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <schemaDirectory>${basedir}/src/main/xsd/</schemaDirectory>
+                            <schemaIncludes>
+                                <schemaInclude>validation-mapping-1.0.xsd</schemaInclude>
+                                <schemaInclude>validation-configuration-1.0.xsd</schemaInclude>
+                            </schemaIncludes>
+                            <bindingDirectory>${basedir}/src/main/xjb/</bindingDirectory>
+                            <bindingIncludes>
+                                <bindingInclude>binding-customization.xjb</bindingInclude>
+                            </bindingIncludes>
+                            <generateDirectory>${project.build.directory}/generated-sources/jaxb/</generateDirectory>
+                            <generatePackage>org.hibernate.validator.internal.xml</generatePackage>
+                            <extension>true</extension>
+                            <enableIntrospection>true</enableIntrospection>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>jaxb2-maven-plugin</artifactId>
                 <executions>
